jenkins.sh

```diff
@@ -2,5 +2,15 @@
 set -x
 set -eu
 
+if [ -n "${GIT_COMMIT-}" ]; then
+  sed -i "s/buildCommit = .*$/buildCommit = \"${GIT_COMMIT}\"/" version.go
```

Comment:
Could we use `-ldflags` to set the value at compile time rather than using `sed`?

I've done this in one of my personal projects:
https://github.com/mattbostock/leavediary/blob/f27014f9be4dffefeead448c67f7f12778730867/Makefile#L1-L7